src/app.js

```diff
@@ -1,21 +1,31 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const getParams = () => {
+  const server = http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const path = url.pathname.slice(1);
+
+    if (path === 'favicon.ico') {
+      return;
+    }
+
+    const parts = path.split('/');
+
+    const query = Object.fromEntries(url.searchParams.entries());
+
+    const data = JSON.stringify({
+      parts,
+      query,
+    });
+
+    res.end(data);
+  });
+
+  server.listen(3000);
```

Comment:
Notify the user that the server is on.
```suggestion
  server.listen(3000, () => console.log("server started"));
```